common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,84 @@
+---
+{
+	"title": "Text-Level Modifiers",
+	"language":"en",
+	"altLangPage":"text-level-modifiers-fr.html",
+	"breadcrumbs":
+		[
+			{
+				"title": "GCWeb home",
+				"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+			}
+		],
+	"secondlevel": false,
+	"dateModified": "2023-07-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.stretched-link</code></dt>
+	<dd>Works in conjunction with position-relative for its stretch limit</dd>
+	<dt><code>.small</code></dt>
+	<dd>Decreases the font size of the element</dd>
+	<dt><code>mark</code></dt>
+	<dd>Highlights the text</dd>
+	<dt><code>.lead</code></dt>
+	<dd>Increases the font size and line height</dd>
+	<dt><code>[class*=text-]</code></dt>
+	<dd>Changes the color of the text</dd>
+	<dt><code>.no-wrap</code></dt>
+	<dd>Prevents the text from wrapping</dd>
+	<dt><code>.fnt-nrml</code></dt>
+	<dd>Resets the font size to the default</dd>
+</dl>
```

Comment:
Create a section for each style/variation. For example

```
<h2>Small (<code>.small</code>)</h2>

<!-- Working example -->
<p class="small">Example of small text style</p>

<!-- Source code -->
<pre><code>&lt;p class="small">Example of small text style&lt;/p></code></pre>
```

For the Stretch link style, you can copy it from this example: https://wet-boew.github.io/wet-boew/demos/utility/utility-en.html#sl

The text color are out of scope here because they will managed through the common "Color (Foreground/Background)".


It is the follow CSS text-* selector that need to be replicated here, each of them need to have their own section.
https://github.com/twbs/bootstrap-sass/blob/6b0ca5025c64f2e5696c8719cfce7148c78d0a50/assets/stylesheets/bootstrap/_type.scss#L91-L100

Reply:
This change don't seem to have been applied yet